docs/annotations.md

```diff
@@ -47,4 +47,4 @@ We recently extended the core API with new features. To enable packages and acti
 2. `web-export`: Applies only to an action. If present, it makes its corresponding action accessible to REST calls _without_ authentication. We call these [_web actions_](webactions.md) because they allow one to use OpenWhisk actions from a browser for example. It is important to note that the _owner_ of the web action incurs the cost of running them in the system (i.e., the _owner_ of the action also owns the activations record).
 3. `require-whisk-auth`: Applies onto to an action. If an action carries the `web-export` annotation, and this annotation is also `true`, the route is only accessible to an authenticated subject. It is important to note that the _owner_ of the web action incurs the cost of running them in the system (i.e., the _owner_ of the action also owns the activations record).
 4. `raw-http`: Applies only to an action in the presence of a `web-export` annotation. If present, the HTTP request query and body parameters are passed to the action as reserved properties.
-
+5. `web-custom-options`: Applies ony to an action. When used with a value of `true`, this annotation enables the ability for a web action to respond to OPTIONS requests with customized headers.
```

Comment:
can you link here to the options section in webactions.md to explain the defaults when the value is false?